tests/test_indexer.py

```diff
@@ -445,8 +447,17 @@ def _get_version(self):
 class PostTestHandler(BaseHTTPRequestHandler):
     _response_code = 200
     _payload = None
+    hmac_secret_key = "ribos0me"
 
     def do_POST(self):
+        HTTPSignatureAuth.verify(requests.Request("POST", self.path, self.headers),
+                                 key_resolver=lambda key_id, algorithm: self.hmac_secret_key.encode())
+        try:
```

Comment:
with `self.assertRaises`

Reply:
This is not a unittest.TestCase subclass.